src/openforms/authentication/contrib/digid_eherkenning_oidc/plugin.py

```diff
@@ -179,16 +182,22 @@ class EHerkenningBewindvoeringOIDCAuthentication(OIDCAuthentication):
 
     def add_claims_to_sessions_if_not_cosigning(self, claim, request):
         # set the session auth key only if we're not co-signing
-        if claim and CO_SIGN_PARAMETER not in request.GET:
-            config = self.config_class.get_solo()
-            request.session[FORM_AUTH_SESSION_KEY] = {
-                "plugin": self.identifier,
-                "attribute": self.provides_auth,
-                "value": claim[config.vertegenwoordigde_company_claim_name],
-                "machtigen": request.session[
-                    EHERKENNING_BEWINDVOERING_OIDC_AUTH_SESSION_KEY
-                ],
-            }
+        if not claim or CO_SIGN_PARAMETER in request.GET:
+            return
+
+        config = self.config_class.get_solo()
+        machtigen_data = request.session[
+            EHERKENNING_BEWINDVOERING_OIDC_AUTH_SESSION_KEY
+        ]
+        request.session[FORM_AUTH_SESSION_KEY] = {
+            "plugin": self.identifier,
+            "attribute": self.provides_auth,
+            "value": claim[config.vertegenwoordigde_company_claim_name],
+            "machtigen": {
+                # TODO So far the only possibility is that this is a BSN.
+                "value": machtigen_data.get(config.gemachtigde_person_claim_name)
```

Comment:
Do I then understand it correctly you always have two attributes?

* DigiD machtigen: BSN of the machtiger and BSN of the gemachtigde
* eHerkenning bewindvoering: KVK number of the company and BSN of the gemachtigde?

I thought that at some point there was also 3 attributes possible: company KVK, BSN of person responsible for company and then BSN of a gemachtigde acting on behalve of the first BSN?

Reply:
I think I told you that would get 2 BSNs in the eHErkenning bewindvoering case, but I was wrong! We get only 1 BSN back (I checked on the ACC environment of DH where bewindvoering works).